### PR TITLE
INDY-1069: move state-depended checks from static to dynamic validation.

### PR DESCRIPTION
**Pull request by sergey-shilov**
Signed-off-by: Sergey Shilov <sergey.shilov@dsr-company.com>

### CODE REVIEW

File: indy_client/test/cli/test_nym.py [hunk 1]
```diff
@@ -361,3 +364,18 @@ def test_send_different_nyms_succeeds_when_batched(
 
     do('send GET_NYM dest={dest}',
         mapper=parameters, expect=CURRENT_VERKEY_FOR_NYM, within=2)
+
+
+def test_nym_resend(looper, sdk_pool_handle, sdk_wallet_steward):
```
Comment (ashcherbakov) [inline indy_client/test/cli/test_nym.py]: This test should not be in the CLI folder

File: indy_node/server/domain_req_handler.py [hunk 2]
```diff
@@ -142,6 +138,10 @@ def _validateNym(self, req: Request):
         origin = req.identifier
         op = req.operation
 
+        s, reason = self.canNymRequestBeProcessed(origin, op)
```
Comment (ashcherbakov) [inline indy_node/server/domain_req_handler.py]: Do we need this call at all? Doesn't `_validateExistingNym` already do similar validation based on `Authoriser`?

File: indy_client/test/cli/test_nym.py (hunk 1 above)
Comment (sergey-shilov) [reply indy_client/test/cli/test_nym.py]: Ok, moved to separate file.

File: indy_node/server/domain_req_handler.py (hunk 2 above)
Comment (sergey-shilov) [reply indy_node/server/domain_req_handler.py]: Yeah, seems like it is double-check, have removed.

**Comment by sergey-shilov** [comment]
Test this please

**Pull request merged**